tools/ci_build/github/download_test_data.py

```diff
@@ -151,10 +151,6 @@ def download_additional_data(build_dir, azure_region):
         print("Starting test data download %s" % url)
         download_and_unzip(args.build_dir, url, models_folder)
 
-        # On windows download additional data
-        if is_windows():
-            download_additional_data(args.build_dir, azure_region)
```

Comment:
I remove this because all the software are preinstalled now, to save job initialization time.